package.json

```
@@ -32,5 +32,11 @@
   },
   "devDependencies": {
     "tree-sitter-cli": "^0.20.7"
-  }
+  },
+  "tree-sitter": [
+    {
+      "scope": "source.kotlin",
+      "file-types": ["kt"]
```

Perhaps add `.kts` so script files get recognized too?

```suggestion
      "file-types": ["kt", "kts"]
```